game-core/src/main/java/games/strategy/triplea/ui/screen/Tile.java

```diff
@@ -15,80 +15,66 @@
 import java.util.List;
 import java.util.PriorityQueue;
 import java.util.Queue;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 /** Responsible for rendering a single map tile. */
 public class Tile {
+  public static final int TILE_SIZE = 256;
+
   private volatile boolean isDirty = true;
-  private volatile boolean isDrawing = false;
+  private AtomicBoolean isDrawing = new AtomicBoolean(false);
 
-  private final Image image;
+  private Image image = Util.newImage(TILE_SIZE, TILE_SIZE, true);
   private final Rectangle bounds;
   private final Object mutex = new Object();
   private final Queue<IDrawable> contents = new PriorityQueue<>();
 
   Tile(final Rectangle bounds) {
     this.bounds = bounds;
-    image = Util.newImage((int) bounds.getWidth(), (int) bounds.getHeight(), true);
```

Comment:
I just noticed "hardcoding" the bounds to TILE_SIZE is no longer neccessary in the final form of this PR 🤔